modules/ivc_champva/lib/ivc_champva/field_transliterator.rb

```diff
@@ -0,0 +1,76 @@
+# frozen_string_literal: true
```

Comment:
@brostk Should this new FieldTransliterator class be in app/services/ rather than lib/ for consistency with Rails conventions?